src/pages/sns.tsx

```diff
@@ -0,0 +1,480 @@
+            <AnimateSpawn
+              el={motion.img}
+              variants={transitions.item}
+              src={BlobGradient}
+              className="absolute pointer-events-none z-[-1]
+              max-w-none
+              w-[1200px]
+              top-[-800px]
+              left-[-800px]
+              sm:w-[1200px]
+              sm:top-[-700px]
+              sm:left-[-800px]
+
+              
+            "
+            />
+          </div>
+          <div
+            className="
+          flex
+          flex-col
+          gap-16 md:gap-0
+          "
+          >
+            <Card
+              title="Community Owned"
+              icon={<IconCommunityOwned aria-hidden="true" />}
+            >
+              <motion.p
+                className="mb-6 tw-paragraph"
+                variants={transitions.item}
+              >
+                Community above all. Fueled by the unique cryptography of the
+                Internet Computer, SNS DAOs empower developers and users to come
+                together to form communities around the full spectrum of
+                applications we use daily. Through an SNS, applications such as
+                social media platforms, B2B enterprise services, play-to-earn
+                games and online news media outlets can now truly become
+                decentralized.
+              </motion.p>
+              <motion.p
+                className="mb-6 tw-paragraph"
+                variants={transitions.item}
+              >
+                All parts of such applications are owned and governed by an SNS
+                DAO. This is in contrasts to existing DAOs whereby governance
+                applies only to smart contract logic, which is just a small part
+                of any web3 application.{" "}
+                <span className="italic">No single entity owns a dapp.</span>
+              </motion.p>
+              <MotionLink
+                className="link-external"
+                href="https://internetcomputer.org/docs/current/tokenomics/sns/sns-intro-tokens"
+                variants={transitions.item}
+              >
+                Learn more
+              </MotionLink>
+            </Card>
+            <Card
+              title="Autonomous"
+              icon={<IconAutonomous aria-hidden="true" />}
+            >
+              <motion.p
+                className="mb-6 tw-paragraph"
+                variants={transitions.item}
+              >
+                Liquid democracy. When a proposal is approved by the community,
+                the SNS implements changes to the code and upgrades itself
+                autonomously. This ensures permisionless and decentralized
+                governance of an SNS DAO.{" "}
+                <span className="italic">
+                  No single entity can stop the process.
+                </span>
+              </motion.p>
+              <MotionLink
+                className="link-external"
```

Comment:
This currently links back to the same page. Should we rather remove the "learn more" for now?